services/web/server/src/simcore_service_webserver/projects/projects_nodes_handlers.py

```diff
@@ -263,6 +266,15 @@ async def stop_node(request: web.Request) -> web.Response:
     """Has only effect on nodes associated to dynamic services"""
     req_ctx = RequestContext.parse_obj(request)
     path_params = parse_request_path_parameters_as(_NodePathParams, request)
+    project_db_api: ProjectDBAPI = request.app[APP_PROJECT_DBAPI]
+
+    save_state = await project_db_api.has_permission(
+        user_id=req_ctx.user_id, project_uuid=path_params.project_id, permission="write"
```

Comment:
If you have read permission on a Study, you should be able to open and stop a node

Reply:
I think that's also taken into consideration. The `write` permission is only used to check if it can save the state and the outputs when closing.

Reply:
@odeimaiz I'd guess that if you have only `read` permission you do not expect the user to be able to change the `outputs` of a service, right?

Currently you can do so, at least for dy-sidecar services. This is never enforced.

Reply:
Great!